dnf-behave-tests/dnf/upgrade-from-noarch.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Upgrade package from noarch to arch
+
+Background: Install package to upgrade
+   Given I use repository "upgrade-from-noarch"
+    When I execute dnf with args "install it-1.0"
+    Then the exit code is 0
+     And DNF Transaction is following
+         | Action        | Package         |
+         | install       | it-1.0-1.x86_64 |
+
+Scenario: Upgrade to rpm from file which is not noarch
+   Given I execute dnf with args "install it.noarch --downloadonly --downloaddir={context.dnf.installroot}/it"
+    When I execute dnf with args "install {context.dnf.installroot}/it/it-2.0-1.noarch.rpm"
```

Comment:
The test passes even on unpatched dnf. I guess you should use the `upgrade` command here, `install` command is not affected by the bug.

Reply:
I am testing again, good catch

Reply:
@m-blaha should be good to go now